angular/core/components/group_page/group_welcome_modal/group_welcome_modal.haml

```diff
@@ -5,8 +5,8 @@
 
   .modal-body
     %p{translate: "group_welcome_modal.{{membershipStatus()}}_info"}
-    .group-welcome-model__video-wrapper
-      %iframe.group-welcome-modal__video{ng-src: "{{helpVideo()}}", frameborder: "0", allowfullscreen=''}
```

Comment:
Reckon that was the error I was seeing before; don't think that's valid haml with the single quotes